src/System.Memory/ref/System.Memory.cs

```diff
@@ -97,14 +97,10 @@ public static partial class MemoryExtensions
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
         public SequencePosition(object @object, int integer) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
```

Comment:
Why did this end up getting removed? I would revert this change, build the ref and src, and then run the generate command from within the ref directory. Please let me know if you are still seeing such side effects/unintended changes.

Reply:
The build target did it.

Reply:
Manually added them back for now.

Reply:
Plz revert.